backend-rust/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to this project will be documented in this file.
 
 Database schema version: 6
 
+### Changed
+
+- Store canonical address as a column on the account
+
 ### Fixed
 
 - Add database migration 6 fixing invalid baker and delegator stake due to missing handling of restake earnings.
```

Comment:
The submodule update points to a PR commit that has meanwhile been closed. Please update the connection to an open PR that can be merged long-term.
https://github.com/Concordium/concordium-rust-sdk/pull/230